### ISSUE TITLE
Minify version not up to date

### ISSUE THREAD

**Issue by Johann-S** [flesler/jquery.preload#2]
Hi,

Since this commit https://github.com/flesler/jquery.preload/commit/bd77be953df3bff1a19c6aed790692ab1b9c93bb the minify version of jquery.preload isn't up to date.

Can I open a PR for this ?

**Comment by flesler** [comment]
Hi Johann, I'll do it, I'll bump the version as well

**Comment by Johann-S** [comment]
thank you :+1:

**Comment by flesler** [comment]
Ok done, no version bump actually, source is still the same.
Thank you for reporting

**Comment by flesler** [comment]
I realized the latest change on source, accepted from a PR didn't have a proper release with version bump. I updated to 1.0.9 and made a new release

**Comment by Johann-S** [comment]
Ok, Just want to know if they are any plan for a futur version of this awesome plugin ?

**Comment by flesler** [comment]
I'm glad that you like it. I do too.
There is no plan to update it further because:
- It doesn't seem to be too widely used
- There are no feature requests
- I don't have any in mind either

It could be that the plugin does what it needs to do, correctly.
I noticed I never published it to NPM like I did for others, I will now that might bump the user base a little.

In short, no, I haven't been updating it for years. If you have a feature in mind and it's not too complex, I might land it asap.
